terraform/README.md

```diff
@@ -0,0 +1,63 @@
+# Bootstrap Flux with Terraform
+
+This example demonstrates how to deploy Flux on a Kubernetes cluster using Terraform
+and the `flux-operator` and `flux-instance` Helm charts.
+
+## Usage
+
+Create a Kubernetes cluster using KinD:
```

Comment:
Maybe mention here that KinD is not a hard requirement, but just a tool we're using to demonstrate this example? (may or may not be obvious depending on the reader)

Reply:
I mentioned in the main readme that KinD can be used for testing.